package.json

```diff
@@ -136,19 +148,43 @@
                 "command": "lean.displayGoal",
                 "key": "ctrl+shift+enter",
                 "mac": "cmd+shift+enter",
-                "when": "editorTextFocus"
+                "when": "editorTextFocus && editorLangId == lean"
             },
             {
                 "command": "lean.batchExecute",
                 "key": "ctrl+shift+r",
                 "mac": "cmd+shift+r",
-                "when": "editorTextFocus"
+                "when": "editorTextFocus && editorLangId == lean"
             }
-        ]
+        ],
+        "menus": {
+            "commandPalette": [
```

Comment:
What effect does this section have?  I can already access all commands from the command palette (the one you can reach with `ctrl+p >` or `ctrl+shift+p`).  Is there another command palette somewhere?